crates/subspace-service/src/sync_from_dsn.rs

```diff
@@ -40,6 +40,8 @@ const CHECK_ALMOST_SYNCED_INTERVAL: Duration = Duration::from_secs(1);
 /// Period of time during which node should be offline for DSN sync to kick-in
 const MIN_OFFLINE_PERIOD: Duration = Duration::from_secs(60);
 
+pub(crate) const LOG_TARGET: &str = "consensus_sync";
```

Comment:
This is exactly the kind of thing we have agreed to remove in the past and exterminated in other parts of the codebase 🙃

Reply:
ohh. Do you remember the why we do not want that ?

I was so difficult for me to sift through all the logs especially the trace events. Having a taget made my life so much easier to debug specific piece of code much faster.

Maybe there was an alternative I'm not aware of and that is the reason we removed it ?

Reply:
It was used in various places in consensus for example and it was really annoying to type `subspace` and see all kinds of things from the whole project. Having crates and modules allowed me to narrow down logs to exactly what I need each time. IMMV :shrug:

Reply:
I'll give this try soon and if it sufficient, i'll give a PR to revert the targeted logs.
Thank you for explaining 👍🏼

Reply:
It is usually more intuitive to do `subspace_service::sync_from_dsn=trace`, even if it is a bit longer.